### PR TITLE
changes after PR 25 Feedback

### PR DESCRIPTION
**Pull request by mserranom**
PR https://github.com/productml/blurr/pull/25

### CODE REVIEW

File: blurr/core/syntax/schema_validator.py [hunk 1]
```diff
@@ -68,24 +68,24 @@ def get_name(self):
     'blurr/core/syntax/dtc_window_schema.yml', validators=VALIDATORS)
 
 
-def _validate_window(dtc_dict, name):
+def _validate_window(dtc_dict: dict, name: str) -> None:
```
Comment (vigbk) [inline blurr/core/syntax/schema_validator.py]: The type is Dict. https://docs.python.org/3/library/typing.html

File: blurr/core/syntax/schema_validator.py [hunk 2]
```diff
@@ -28,37 +28,37 @@ class DataType(Validator):
...
-        return "Identifier"
+    def get_name(self) -> str:
+        return 'Identifier'
 
 
 class Expression(Validator):
     TAG = 'expression'
 
-    def _is_valid(self, value):
+    def _is_valid(self, value) -> bool:
```
Comment (prashastk) [inline blurr/core/syntax/schema_validator.py]: value should be str?

File: blurr/core/syntax/schema_validator.py (hunk 1 above)
Comment (mserranom) [reply blurr/core/syntax/schema_validator.py]: fixed

File: blurr/core/syntax/schema_validator.py (hunk 2 above)
Comment (mserranom) [reply blurr/core/syntax/schema_validator.py]: fixed

File: blurr/core/syntax/schema_validator.py (hunk 1 above)
Comment (mserranom) [reply blurr/core/syntax/schema_validator.py]: fixed

**Pull request merged**